Library/Homebrew/cask/lib/hbc/cli/doctor.rb

```diff
@@ -60,12 +60,12 @@ def check_install_location
       def check_staging_location
         ohai "Homebrew-Cask Staging Location"
 
-        path = Pathname.new(user_tilde(Caskroom.path.to_s))
+        path = Caskroom.path
 
         if !path.exist?
-          add_error "The staging path #{path} does not exist."
+          add_error "The staging path #{user_tilde(path)} does not exist."
         elsif !path.writable?
-          add_error "The staging path #{path} is not writable by the current user."
+          add_error "The staging path #{user_tilde(path)} is not writable by the current user."
         end
 
         puts path
```

Comment:
This should also use `user_tilde`.

Reply:
Addressed. Rebased on master and force-pushed.